README.md

```diff
@@ -1,37 +1,133 @@
 # Bonita BPM documentation resources
 
-This repository contains the new Bonita BPM documentation site & tools.  
-It is using markdown to create documentation content.
+This repository contains the new Bonita BPM documentation site content.
+It uses [Markdown](https://help.github.com/categories/writing-on-github/) to create documentation content.
+
+
+## View content
+
+### Simple view
+
+Using the [Github Markdown Format](https://help.github.com/categories/writing-on-github/) format allows to check the documentation directly on the GitHub repository website.
+
+So a simple way to view documentation content is to browse the `md` folder on GitHub website.
+
+### Advanced view
+
+A more industrialized process is possible.  
+The project uses JavaScript scripts to manage and deploy content.
+Therefore, you must first install your favorite `node` version to be able to build the project (`nvm` is the best place to start).
+
+Then run in the root folder:
+
+    npm install
+
+Then run
+
+    npm run watch
+
+A web server will be launched locally. It converts md files to html (currently without any style).
+A livereload is also set up and allows any change to the `md` files to reload its html version in the browser.
+
 
 ## Contribute
 
+### Update existing pages
+
+In this case, a simple edit of the _.md_ file in the appropriate branch (one per version) is enough, the build process will do the rest.
+
+<u>Example</u>: To edit the **actors.html** page in the 7.3 version, checkout the **versions/7.3** branch and edit the `md/actors.md` file.
```

Comment:
we should proceed the same way than with development project : rebase master branch on previous branch